xmake/rules/c++/modules/clang/support.lua

```diff
@@ -225,6 +217,7 @@ end
 
 -- get clang version
 function get_clang_version(target)
+
```

Comment:
remove it

Reply:
how many line before i need to add the empty line ? i was going for >=5 lines

Reply:
>= 20